extensions/typescript-language-features/src/typescriptServiceClient.ts

```diff
@@ -140,10 +140,13 @@ export default class TypeScriptServiceClient extends Disposable implements IType
 		this._configuration = TypeScriptServiceConfiguration.loadFromWorkspace();
 		this.versionProvider = new TypeScriptVersionProvider(this._configuration);
 		this.pluginPathsProvider = new TypeScriptPluginPathsProvider(this._configuration);
-		this._versionManager = this._register(new TypeScriptVersionManager(this.versionProvider, this.workspaceState));
+		this._versionManager = this._register(new TypeScriptVersionManager(this._configuration, this.versionProvider, this.workspaceState));
 		this._register(this._versionManager.onDidPickNewVersion(() => {
 			this.restartTsServer();
 		}));
+		this._register(this._versionManager.onEnteredPromptWorkspaceTsdkState(() => {
```

Comment:
Looks like this event is now only being used to call back into the version manager, so let's removed it and just have the version manager call `promptUseWorkspaceTsdk`